R/oc_check_query.R

```diff
@@ -0,0 +1,192 @@
+#' Check OpenCage query arguments
+#'
+#' Function that checks the query arguments passed to OpenCage
+#'
+#' @param limit The maximum number of results that should be returned. Integer
+#'   values between 1 and 100 are allowed.
+#' @inheritParams oc_forward
+#'
+#' @keywords internal
```

Comment:
since it's internal why not add a `@noRd` tag?

Reply:
Because then there is no Rd file, which defeats the point of `@inheritParams oc_forward`. If you think a help file it too much, I think I'd rather delete the roxygen comments?